hyperparameter_tuning/model_tuning_for_distributed_training/hyperparameter_tuning_for_distributed_training.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>External links often break without any notice - use the same dataset from our public S3 bucket s3://sagemaker-sample-files/datasets/tabular/uci_bank_marketing/bank-additional-full.csv instead </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks So much!